### PR TITLE
Added mysql and postgres support to qt build

### PR DESCRIPTION
**Pull request by kberryUSGS**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

This just adds mysql and postgres support to the qt build.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -163,6 +163,8 @@ requirements:
     - nspr                               # [unix]
     - nss                                # [unix]
     - sqlite
+    - mysql                              # [linux]
+    - postgresql                         # [linux]
     - zlib
```
Comment (mingwandroid) [inline recipe/meta.yaml]: what about macOS here?

Comment (kberryUSGS) [reply recipe/meta.yaml]: Good point, thanks!

**Comment by jlaura** [comment]
@ccordoba12 @gillins @mingwandroid @msarahan @ocefpaf @stuarteberg 
Possible to get a review on this PR? We would love to cut over from our snowflake qt build to the official conda-forge one. Much appreciated!

**Comment by isuruf** [comment]
Windows build failure here.

**Comment by kberryUSGS** [comment]
@isuruf The windows build didn't fail this time, but it timed out. Should I re-run this, or is it expected?

**Comment by isuruf** [comment]
That's expected

**Comment by kberryUSGS** [comment]
@isuruf I see that the mac build for this went up on anaconda cloud a bit over a month ago. Would it be possible for a Linux build to be uploaded, too?

**Pull request merged**